lib/modules/dashboard.js

```diff
@@ -1,4 +1,5 @@
 import { $ } from '../vendor';
+import { go } from '../core/init';
```

Comment:
Nit: use `import * as Init` to distinguish it from utils and other free functions (which can be used freely, whereas I want to discourage the use of `Init` in modules except where necessary)